### DISCUSSION TITLE
git clone can drop all to opt

### DISCUSSION THREAD

**Discussion by jjrbfi** [grotius-cnc/hal-core#10] [Q&A]
We can drop files to /opt/halcore with out using cp after git clone. (I made changes on that but where re-changed)

$ git clone https://github.com/grotius-cnc/hal_core.git **/opt/hal-core**

**Comment by grotius-cnc** [comment]
ok, nice i will try it out.

**Comment by grotius-cnc** [comment]
it works! nice.

**Reply by jjrbfi** [reply]
I did that change but was re-changed.
I will close/lock this.
